docs/conversion/s2u_features/S2U-14-mysql.sql

```diff
@@ -0,0 +1,3 @@
+-- S2U-14 --
+ALTER TABLE SAM_PUBLISHEDITEM_T ADD CANCELLED INT(11) DEFAULT 0 NOT NULL;

```

Comment:
Should we use a TINYINT for this using only 1 byte? We cover probably all the cases

Reply:
I think the name should be 'CANCELLATION', could you confirm?